sdk/file_sdk/Common/file_format.cpp

```diff
@@ -0,0 +1,25 @@
+#include "file_format.h"
+
+namespace mip {
+namespace file {
+
+FileFormat::FileFormat(
+    std::shared_ptr<IStream> file,
+    const string& extension)
+  : mFile(file),
+    mExtension(extension)
+{
+}
+
+const vector<mip::Tag>& mip::file::FileFormat::GetTags()
+{
+  return mTags;

```

Comment:
We talked about that GetTags will read tags from file it the tags were not set or were not read previously.

---
In reply to: [134442532](https://github.com/AzureAD/rms-sdk-for-cpp/pull/133#discussion_r134442532) [](ancestors = 134442532,134435478)